dispatcher/backend/src/routes/users/user.py

```diff
@@ -129,6 +130,8 @@ def patch(self, token: AccessToken.Payload, username: str):
     @url_object_id("username")
     def delete(self, token: AccessToken.Payload, username: str):
         # delete user
+        if get_role_for(Users().find_one({"username":username}).get("scope")) == "worker":
```

Comment:
Inserting new code in between a comment and the code that's related to it is misleading.

Reply:
You should use two steps so that the workers delete is only happening after the user is deleted

Reply:
This will crash if there is no User with that username as `find_one` will return `None`

Reply:
You didn't respect the black formatting (`pip install black` then `black <yourfile`)